### PR TITLE
object_zl2, object_zl2_anime1 & object_zl2_anime2 OK

### PR DESCRIPTION
**Pull request by AngheloAlf**
Decompiled the objects of adult Zelda, used by `z_en_eg`, `z_en_zl2` and. `z_en_zl3`. I also did a bit of renaming of those actors.

### CODE REVIEW

**Review by fig02 — APPROVED** [review]
looks good, but you can remove the gitkeep files. we only need one in the asset folders root. same goes for your other open prs if they have them

**Comment by AngheloAlf** [comment]
Done

File: assets/xml/objects/object_zl2.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<Root>
+    <File Name="object_zl2" Segment="6">
+        <!-- Zelda 2 skeleton -->
+        <Skeleton Name="gZelda2Skel" Type="Flex" LimbType="Standard" Offset="0x10D70"/>
+
+        <!-- Zelda 2 eyes textures -->
+        <Texture Name="gZelda2EyesPalette" OutName="zelda_2_eyes_palette" Format="rgb5a1" Width="16" Height="16" Offset="0x2B00"/>
```
Comment (Roman971) [inline assets/xml/objects/object_zl2.xml:7]: I don't believe we have discussed a suffix for this yet, and "palette" seems a bit long, so we should probably bring it up in discord and see what people think (something like "pal" might be good enough but it's worth a discussion either way)

Comment (EllipticEllipsis) [reply assets/xml/objects/object_zl2.xml:7]: Given that we'll need to support PAL/NTSC versions of objects, that's probably the one abbreviation we don't want to use!

Comment (fig02) [reply assets/xml/objects/object_zl2.xml:7]: its important to consider, but i think context helps. and fwiw they originally used the pal abbreviation as well (see skyboxes)

Comment (fig02) [reply assets/xml/objects/object_zl2.xml:7]: We should get back to this discussion.
I still think `Pal` is fine and doesnt conflict with the region abbreviation at all (both context and capitalization make it clear imo), but if more people want TLUT then we should settle on what the majority wants

Comment (fig02) [reply assets/xml/objects/object_zl2.xml:7]: guess this conversation isnt really going to progress more, so TLUT is probably what the majority of people were favoring

**Comment by Roman971** [comment]
I could be wrong, but I think the idea was to have TLUT as the suffix (all uppercase)

**Comment by fig02** [comment]
Im down with either. currently we do things like `Id` instead of `ID` to follow camel case, so idk if one makes more sense over the other
Worth noting the gfx macro does it in all uppercase

**Comment by AngheloAlf** [comment]
I don't like this suffix being `TLUT` in all caps, but I think we should use it like we use `DL` for display lists.

**Review by Roman971 — APPROVED** [review]

**Pull request merged**